source/assets/stylesheets/partials/_blog.scss

```diff
@@ -0,0 +1,54 @@
+.blog_heading {
+  text-align: center;
+}
+
+.narrow_container {
+  max-width: 700px;
+  margin: 50px auto 0 auto;
+}
+
+.article-preview {
+  border-bottom: 1px solid $base-border-color;
```

Comment:
These borders look too dark. Let's use the color used on the [Doctrine page](https://euroteamoutreach.org/doctrine/).